docs/Reference/Rest_API/API-Objects.md

```diff
@@ -7,14 +7,24 @@ The following objects are returned by Teku REST API methods.
 
 ## Beacon block
 
-Returned by [`beacon/block`](https://pegasyseng.github.io/teku/stable/#operation/getBeaconBlock).
+Returned by [`beacon/block`](https://pegasyseng.github.io/teku/#operation/getBeaconBlock).
+
+| Field                | Type                | Description                                         |
+|----------------------|:-------------------:|-----------------------------------------------------|
+| **root**             | Byte32              | Block root of the beacon block.                     |
+| **beacon_block**     | Object              | Object containing the block details.                |
+
+The `beacon_block` object contains a `message` object with the following signed beacon block
```

Comment:
how does this relate to the "Beacon block body" section below?

Reply:
The `body` field in the table is related to the "Beacon block body". I'll update the description in "Beacon block body to call out that relationship.